.github/workflows/testing.js.yml

```diff
@@ -176,20 +177,18 @@ jobs:
         run: rm -rf frank-runner/build/apache-tomcat-9.0.56/logs/catalina.out
 
       - name: Let Frank!Runner do its downloads
-        run: pnpm run installFrankRunnerEnv
-        working-directory: ladybug-frontend
+        run: cd frank-runner && ./env.sh > installFrankRunnerEnv.log
```

Comment:
Why did you remove the working directory here but kept it in other jobs?

Reply:
Is geen goede reden voor. Ik verander het.